### PR TITLE
Second modal

### PR DESCRIPTION
**Pull request by Aselshine**
Issue #:
<a href="https://trello.com/b/oudZuA4c/workoutapp">Trello</a>

### PR type
<br>

<p>Please check the type of change your PR introduces:</p>

- [] Bugfix
- [x] Feature
- [] Code style update (formatting, renaming)
- [] Refactoring (no functional changes, no api changes)
- [] Build related changes
- [] Other (please describe):

### Screenshots
<br>

<h4>before:</h4>

<h4>after:</h4>
Снимок

### Does this introduce a breaking change?
<br>

- [] yes (please describe):
- [] no

### Pull request checklist
<br>

<p>Please check if your PR fulfills the following requirements:</p>

- [] Tests for the changes have been added (for bug fixes / features)
- [] Accessibility support has been tested
- [] Responsive layout and functionality have been tested
   - [] desktop and mobile view responsiveness are supported
   - [] no loss of functionality or content when zoomed in to 400%
   - [] no horizontal scrolling when zoomed in to 400% (only vertical scrolling, exception: DataGrids)
- [] avoids using many levels of prop drilling
- [] check console on errors and warnings.

<hr>

### CODE REVIEW

File: src/components/SecondModal/SecondModal.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const SecondModal = () => {
+  const [woman] = useState(true);
+  const [man] = useState(true);
+  const handleChange = () => {};
+  return (
+    <div className={styles.secondModal}>
+      <img className={styles.arrow} src="/Images/left-arrow.png" alt="" />
+      <img className={styles.close} src="/Images/close.png" alt="" />
+      <h1>Ваш пол?</h1>
+      <p>Отвечая на наши вопросы Вы поможете нам составить Ваш персональный план тренировок</p>
```
Comment (mdktva) [inline src/components/SecondModal/SecondModal.jsx:12]: You should do translation in english as well when it's necessary

File: src/components/SecondModal/SecondModal.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      <div className={styles.manIcon}>
+        <input
+          type="checkbox"
+          id={styles.checkbox2}
+          value={man}
+          onChange={() => handleChange("man")}
+        />{" "}
+        мужской <br />
+        <label htmlFor="checkbox2">
+          <img src="/Images/man1.png" alt="" />
```
Comment (mdktva) [inline src/components/SecondModal/SecondModal.jsx:35]: you should fill alt="" with text

File: src/components/SecondModal/SecondModal.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import styles from "./SecondModal.module.scss";
+const SecondModal = () => {
+  const [woman] = useState(true);
+  const [man] = useState(true);
+  const handleChange = () => {};
+  return (
+    <div className={styles.secondModal}>
+      <img className={styles.arrow} src="/Images/left-arrow.png" alt="" />
```
Comment (mdktva) [inline src/components/SecondModal/SecondModal.jsx:9]: fill alt

File: src/components/SecondModal/SecondModal.jsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      <img className={styles.close} src="/Images/close.png" alt="" />
```
Comment (mdktva) [inline src/components/SecondModal/SecondModal.jsx:10]: here as well

File: src/components/SecondModal/SecondModal.jsx (hunk 3 above)
Comment (Aselshine) [reply src/components/SecondModal/SecondModal.jsx]: done

File: src/components/SecondModal/SecondModal.jsx (hunk 2 above)
Comment (Aselshine) [reply src/components/SecondModal/SecondModal.jsx]: done

File: public/locales/ru/translation.json [hunk 5]
```diff
@@ -1,5 +1,18 @@
 {
-  "header":{
-    "title": "Заголовок"
+    "header":{
+      "title": "Header",
+      "btn": "Can"
+    },
+    "register":{
```
Comment (russllan) [inline public/locales/ru/translation.json:6]: please make the attribute names the same

**Comment by russllan** [comment]
1) please test for errors in the console
2) please take a screenshot of your project
3) in the images folder create another folder with the appropriate name

File: public/locales/ru/translation.json (hunk 5 above)
Comment (Aselshine) [reply public/locales/ru/translation.json]: in both en and ru folders, attribute changed into reg

**Comment by Aselshine** [comment]
Снимок10
Снимок11

File: src/components/SecondModal/SecondModal.jsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+      <img className={styles.close} src="Images/SecondModalIcons/close.png" alt="close" />
+      <h1>{t("reg.title")}</h1>
+      <p>{t("reg.text")}</p>
+
+      <div className={styles.manIcon}>
+        <input
+          type="checkbox"
+          id={styles.checkbox2}
+          value={man}
+          onChange={() => handleChange("man")}
```
Comment (russllan) [inline src/components/SecondModal/SecondModal.jsx:21]: send the entered data to redux please

Comment (Aselshine) [reply src/components/SecondModal/SecondModal.jsx]: Снимок16
Снимок17
Снимок18

**Comment by russllan** [comment]
update screenshots. And so all is well

**Review by russllan — APPROVED** [review]

**Pull request merged**